test/unit/auth/test_jwt_auth.py

```diff
@@ -25,6 +25,11 @@ def jwt_algorithm(request):
     return request.param
 
 
+@pytest.fixture(params=('key_id_1', 'key_id_2'))
```

Comment:
No, I don't think it gives us much. I did this just to be consistent with other fixtures in this test module.